fortnox_huvudbok_processing/projects_2023.json

```diff
@@ -0,0 +1,32 @@
+{
+    "992100": "Medlemmar",
+    "992150": "Insamling från privatpersoner",
+    "992200": "Styrelse",
+    "992300": "Verksamhetschef/ledning",
+    "992400": "Anställda",
+    "992401": "Utbildning",
+    "992500": "Ansökningar",
+    "992700": "Administration",
+    "992800": "Personalkostnader",
+    "213102": "Stöd för innehållspartnerskap",
+    "213123": "Wikidata för auktoritetskontroll",
+    "213203": "Internationell upphovsrätt",
+    "215205": "Sänkta trösklar för WikiGap",
+    "223103": "MP - Pattypan 2022",
+    "223121": "MP - Nordiska Museet 2022",
+    "225206": "Wikipedia för hela Sverige",
+    "231100": "Innehållspartner 2023",
+    "231200": "Beslutsfattare 2023",
+    "231214": "Rätten till kunskap i Sverige",
+    "231210": "Politiska beslutsfattare 2023",
+    "231500": "Internationella innehållspartner 2023",
+    "231503": "Kapacitetsbyggnad i Innehållspartnerskapshubben",
+    "232100": "Läsare 2023",
+    "232500": "Nya användare 2023",
+    "233100": "Befintliga gemenskaper 2023",
+    "233200": "Nya gemenskaper 2023",
+    "234500": "Föreningsutveckling 2023",
+    "234501": "Föreläsningar 2023",
+    "234400": "Internationella fri kunskapsrörelsen 2023",
+    "234501": "MP - Föreläsningar 2023"
```

Comment:
The number is the same as on two rows above. Guessing this is an unintentional duplicate.